examples/connext_dds/compression/resource/text_payload.txt

```diff
@@ -0,0 +1,129 @@
+Lorem ipsum dolor sit amet consectetur adipiscing elit sagittis etiam, egestas
```

Comment:
You could put this in the top level folder maybe `resources/messages`. It seems like a generic concept.

Reply:
done